python_modules/dagster/dagster_tests/core_tests/storage_tests/utils/event_log_storage.py

```diff
@@ -525,6 +525,33 @@ def _solids():
 
         assert _event_types(out_events) == _event_types(events)
 
+    def test_get_logs_for_run_cursor_limit(self, storage):
+        @solid
+        def return_one(_):
+            return 1
+
+        def _solids():
+            return_one()
+
+        events, result = _synthesize_events(_solids)
+
+        for event in events:
+            storage.store_event(event)
+
+        out_events = []
+        cursor = -1
+        fuse = 0
+        while fuse < 50:
```

Comment:
is this just a safeguard in case of error (e.g. not loop forever)?

Reply:
yea